pkg/telemetry/phonehome/client_config.go

```diff
@@ -19,6 +24,12 @@ var (
 // services tenant ID. The value of the label becomes the group ID if not empty.
 const TenantIDLabel = "rhacs.redhat.com/tenant"
 
+// Interceptor is a function which will be called on every API call if none of
+// the previous interceptors in the chain returned false.
```

Comment:
Here there is no chain and no workflow logic, so I'd move this comment to:
* `track()` describing the workflow logic
* `AddInterceptorFunc()` saying that the order in which interceptors are added might matter

Here I'd describe when an `Interceptor` instance is supposed to return `true` and when `false`.